firstvoices/backend/views/join_request_views.py

```diff
@@ -196,7 +204,12 @@ def approve(self, request, site_slug=None, pk=None):
                 join_request, JoinRequestStatus.APPROVED, request.user
             )
 
-        # notify user here, see FW-5077
+        subject = f"Your FirstVoices account has been approved to join {join_request.site.title}."
+        message = (
+            f"Thank you for your interest in joining the FirstVoices site {join_request.site.title}.\n\n"
+            f"Your account has been approved with the {role.label} role and you can now log in to the site.\n\n"
```

Comment:
Minor: the site uses "sign in" rather than "log in" so probably this message should as well